ledger/src/shred/shred_code.rs

```diff
@@ -13,9 +13,6 @@ use {
     static_assertions::const_assert_eq,
 };
 
-const_assert_eq!(MAX_CODE_SHREDS_PER_SLOT, 32_768);
-pub const MAX_CODE_SHREDS_PER_SLOT: usize = MAX_DATA_SHREDS_PER_SLOT;
```

Comment:
I think it would be better to either keep this or to create a new const with a name that implies it is intended for both data and coding shred limits. Using `MAX_DATA_SHREDS_PER_SLOT` to bound coding shreds is confusing

Reply:
Agreed with Brennan on this one - if we keep the "separate" constant we could even mention the SIMD above the line where we do `DATA_SHREDS_PER_SLOT = CODE_SHREDS_PER_SLOT`.

> Small steps on the way to clean up the Shredder API

Your PR description mentions this - this doesn't directly touch anything in shredder. Can you elaborate on this, maybe some stuff that you have lined up for subsequent PR's ?

Reply:
Cool, thanks for sharing